src/types/fhir-objects.d.ts

```diff
@@ -15,7 +15,21 @@ declare module 'fhir-objects' {
   export type Practitioner = fhir.Practitioner | R4.IPractitioner;
   export type Procedure = fhir.Procedure | R4.IProcedure;
   export type Resource = fhir.Resource | ResourceR4;
-  export type Bundle = fhir.Bundle | R4.IBundle;
   export type ServiceRequest = fhir.ProcedureRequest | R4.IServiceRequest;
   export type MedicationRequest = fhir.MedicationRequest | R4.IMedicationRequest;
+  export type Bundle = R4.IBundle & { entry: BundleEntry[] };
```

Comment:
I'm a little confused about what's going on here. Is the intent here just to make certain fields on these types mandatory as opposed to optional?

Reply:
Yes all of the `& {}` I added are to make the fields mandatory